packages/frontend-v2/components/index.ts

```diff
@@ -1 +1 @@
-export * from "./Error";
+export * from "./Error";
```

Comment:
you can install the prettier extension (by "prettieR", has 21m downloads) and then enable "format on save" in preferences, and this should get fixed automatically